crates/library_config/Cargo.toml

```diff
@@ -1,18 +1,21 @@
 [package]
 name = "library-config"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Maybe `0.1.1` is more appropriate, WDYT @szegedi ?

Reply:
since this is basically a binding for libdatadog, and we updated libdatadog to a higher minor, I'd bump the minor.